src/style.css

```diff
@@ -36,24 +40,28 @@ ul {
   gap: 20px;
 }
 
-img {
+.moyo-header-logo {
   width: 40px;
   height: 40px;
   flex-shrink: 0;
 }
 
-a {
+.nav_link {
   display: flex;
   text-decoration: none;
   color: #000;
   line-height: 60px;
 }
 
-a[data-qa='hover']:hover {
+.nav_link:hover {
+  color: var(--blue-color);
+}
+
+[data-qa='hover']:hover {
```

Comment:
this is also nav link, you don't need to style it by data attr.

Reply:
Ok